apps/component-generator/component-templates/ComponentTemplate/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui-react-native/component-name",
+  "version": "0.15.8",
+  "description": "add component-description",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
```

Comment:
Is the intent that this is a template file or to ship the component template as a package?

Is this is meant to be a template then any publish-related lines should be removed & this package should be made private so that the template isn't considered a new component/package that we're shipping.

Reply:
Yes, it should be a template. Thanks for catching this